packages/instrumentor/instrument.test.ts

```diff
@@ -95,8 +98,11 @@
 	const oldFuzzer = globalThis.Fuzzer;
 	// @ts-ignore
 	globalThis.Fuzzer = {
-		incrementCounter: () => {
-			// ignore
+		// @ts-ignore
+		coverageTracker: {
+			incrementCounter: (edgeId: number) => {
```

Comment:
Warning that the parameter is not needed, can be removed.

Reply:
Removed.

Reply:
The could will not compile if we remove this ignore directive